NetKet/Graph/hypercube.hpp

```diff
@@ -218,30 +218,43 @@ class Hypercube : public AbstractGraph {
   static std::vector<std::vector<int>> BuildSymmTable(int const L,
                                                       int const n_dim, bool pbc,
                                                       int const n_sites) {
-    if (!pbc) return {};
-    assert(L >= 2 && n_dim >= 1 &&
-           "Bug! This should be the case by construction.");
-    std::vector<std::vector<int>> symm_table;
```

Comment:
But what's wrong with the rest of the implementation?

Reply:
I am not sure, but your implementation gives different permtables for square or cubic lattices. I think you are not considering all the possible translation vectors, but I am not sure